src/mount/mastercomm.cc

```diff
@@ -374,6 +375,21 @@ static bool fs_threc_send_receive(threc *rec, bool filter, PacketHeader::Type ex
 const uint8_t* fs_sendandreceive(threc *rec, uint32_t expected_cmd, uint32_t *answer_leng) {
 	// this function is only for compatibility with Legacy code
 	sassert(expected_cmd <= PacketHeader::kMaxOldPacketType);
+
+#ifdef _WIN32
+    // Code specific to Windows
+    if (masterServerIsDisconnected) {
+        const uint8_t defaultAnswer[] = {
```

Comment:
I agree with Baldor, the numbers should be explained in some way.
Also: the variable should be static to avoid initialization every time or even a global constexpr.

Reply:
Done